nupic/torch/modules/sparse_weights.py

```diff
@@ -112,18 +122,20 @@ class SparseWeights(SparseWeightsBase):
 
     :param module:
       The module to sparsify the weights
-    :param weight_sparsity:
-      Pct of weights that are allowed to be non-zero in the layer.
+    :param sparsity:
+      Pct of weights that are zero in the layer.
```

Comment:
Could be good to change this to:
"Fraction (b/w 0 and 1) of weights that are zero in the layer."

As the range we allow is technically not a percentage.